tests/integ/test_tuner.py

```diff
@@ -639,6 +633,7 @@ def test_tuning_tf_script_mode(sagemaker_session, cpu_instance_type, tf_full_ver
         tuner.wait()
 
 
+# TODO: evaluate skip mark and default framework_version 1.11
```

Comment:
we can delete this test - looks like I forgot about it when I was cleaning up TF